scripts/docker/install_mysql.sh

```diff
@@ -51,6 +51,7 @@ if [[ ${INSTALL_MYSQL_CLIENT:="true"} == "true" ]]; then
     rm -rf "${GNUPGHOME}"
     apt-key list > /dev/null
     echo "deb http://repo.mysql.com/apt/debian/ stretch mysql-5.7" | tee -a /etc/apt/sources.list.d/mysql.list
+    echo "deb http://repo.mysql.com/apt/debian/ stretch mysql-8.0" | tee -a /etc/apt/sources.list.d/mysql.list
```

Comment:
Wy PR baby want to switch to buster from stretch since we are using buster as base.

Reply:
oh yes